src/content/roadmap/_index.en.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Roadmap"
+date = 2020-02-19T20:24:34+01:00
+weight = 100
+pre = "<b>5. </b>"
++++
+
+This a preliminary community roadmap, it's not written in stone, but it can serve
+as a guideline about what's ahead.
+
+Please see details of previous releases [here](../releases)
+
+### v0.1.1
+
+* Stabilization of new lighthouse related features and deployment
```

Comment:
May be making a lighthouse a default Openshift plugin can also be a part of this.

Reply:
I think we should not do that, because it means it will break the OperatorVersion and upgrade of the cluster, we need to make it may be more visible on the guides. We can do that when we remove the kubefed dependency and fix the ClusterVersion operator problem IMO.

Reply:
But it's not something we control from our versioning, I can make a reference to it